Readme.md

```diff
@@ -1,4 +1,8 @@
 # webvi-examples
+- Please use all of the default namespaces and directory names when building the Web Application in each example
+- Fork or branch this repo and work on a single example in a single branch at a time
+- Draft the readme for each example as you go. The Readme doesn't need to be perfect, but there must be something before we can pull it in.
+
```

Comment:
Can you point to an example README for the example READMEs?

ie what sections should they have?

setup
dependencies, ie NXG, LV2016, etc
overview
description
usage
expected behavior